standards/SPIFFE_Trust_Domain_and_Bundle.md

```diff
@@ -0,0 +1,290 @@
+
+The manner in which this is done is dependent upon the type of authentication and validation used. For instance, if using Web PKI, then an operator-supplied `<trust_domain, endpoint_address>` tuple is sufficient. The trust domain is explicitly bound to the endpoint address through the supplied tuple, and (per standard Web PKI secure naming) the endpoint address is equivalent to the identity of the serving workload. 
+
+This can be contrasted to the use of SPIFFE authentication, where the identity of the workload is not the same as the endpoint address. In this case, a different parameter (the SPIFFE ID of the bundle endpoint) is required. The trust domain can be implicitly extracted from the provided SPIFFE ID. Concretely, this translates into an operator-supplied tuple of `<trust_domain, spiffe_id, endpoint_address>` or simply `<spiffe_id, endpoint_address>`.
+
+Please see the [Authenticating the Bundle Endpoint](#52-authenticating-the-bundle-endpoint) section and the [Bundle Endpoint Authentication Examples](#appendix-b-bundle-endpoint-authentication-examples) appendix item for more information.
+
+### 6.2. SPIFFE Bundle Refresh Hint
+SPIFFE bundles include an optional `refresh_hint` field meant to indicate the frequency at which consumers should attempt to refresh their copy of the bundle. This value has a clear impact on how quickly keys can be rotated, but it also impacts how quickly keys can be redacted. Refresh hint values should be carefully chosen with this in mind.
+
+Since this field is not mandatory, it is possible to encounter SPIFFE bundles that do not have a `refresh_hint` set. In this case, a client has the option of using a suitable interval by examining SVID validity periods. It should be acknowledged that omitting a `refresh_hint` will likely impact the ability of a trust domain to rapidly revoke compromised keys. Clients should default to a relatively low (e.g. five minutes) refresh interval to be able to retrieve updated trust bundles in a timely manner.
+
+### 6.3. Using Web PKI on a SPIFFE Bundle Endpoint
+Web PKI provides a convenient way to authenticate remote SPIFFE bundle endpoints as publicly trusted authorities can negate the need for an initial out-of-band bundle exchange (as is the case when using SPIFFE authentication). While largely reliable, there are some tradeoffs and drawbacks that should be considered before its use.
+
+The first and most obvious is that you are assuming trust in the public certificate authorities and in their fulfillment procedures. Any public CA can sign a certificate for the domain name where your bundle endpoint is hosted that will appear valid. While this may be an unacceptable risk to some, most users are OK with it given the convenience it provides and the historical stability of the public CA infrastructure.
```

Comment:
Consider changing the second sentence to improve readability: Publicly trusted CAs have the technical ability to sign certificates which will be broadly trusted and therefore could, in theory, sign a valid certificate for the host name used to serve your bundle endpoint.

Nit: third sentence: consider s/OK/okay/.